src/main/java/ch/ivyteam/ivy/maven/engine/deploy/dir/FileDeployer.java

```diff
@@ -128,8 +127,7 @@ private static void wait(Supplier<Boolean> condition, long duration, TimeUnit un
           throw new TimeoutException("Operation reached timeout of " + duration + " " + unit);
         }
         Thread.sleep(100);
-      } catch (InterruptedException ex) {
-      }
+      } catch (InterruptedException ex) {}
```

Comment:
bad exception handling here.
catching interrupted exception should set the interrupted flag on the curren thread again.